packages/tools/devtools/devtools-core/src/data-visualization/VisualTree.ts

```diff
@@ -74,6 +74,11 @@ export interface VisualNodeBase {
 	 * Inside of the porperty is an array of possible EditTypes to inform devtools-view to only show the corresponding edit options for the types allowed
 	 */
 	editProps?: { editTypes?: EditType[] };
+
+	/**
+	 * Visual Tree data rendered in the tooltip.
+	 */
+	tooltipContents?: string | Record<string, string>;
```

Comment:
I still think this should take a visual tree, rather than simple record data. Let's at least leave a TODO comment about this in a `@privateRemarks` block.